src/scores/probability/crps_impl.py

```diff
@@ -855,8 +858,7 @@ def crps_for_ensemble(
     ensemble_member_dim1 = scores.utils.tmp_coord_name(fcst)
 
     # calculate forecast spread contribution
-    fcst_copy = fcst.rename({ensemble_member_dim: ensemble_member_dim1})  # type: ignore
-
+    fcst_copy = fcst.rename({ensemble_member_dim: ensemble_member_dim1})  # type: ignoreßß
```

Comment:
Looks like a couple of symbols got accidentally pasted after `type: ignore`. Please strip them out.

Reply:
Weird - thanks for picking up on that